estate/models/house_tag.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+class house_tag(models.Model):
+    _name = 'estate.house_tag'
```

Comment:
```suggestion
    _name = 'estate.house.tag'
```
The used convention in odoo is using words seperated by dots.
Using an underscore is not wrong and it's used in some cases but it's not common